examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,124 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example re-creates figures from a paper from the Lalor group which
+demos the mTRF toolbox in Matlab [1]_. We will show how the
+:class:`mne.decoding.ReceptiveField` class can perform a similar function
+along with :mod:`sklearn`. We'll fit a linear encoding model using the
+continuously-varying speech envelope to predict activity of a 128 channel
+EEG system.
+
+References
+----------
+.. [1] Crosse, M. J., Di Liberto, G. M., Bednar, A. & Lalor, E. C. (2016).
+       The Multivariate Temporal Response Function (mTRF) Toolbox:
+       A MATLAB Toolbox for Relating Neural Signals to Continuous Stimuli.
+       Frontiers in Human Neuroscience 10, 604. doi:10.3389/fnhum.2016.00604
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+from mne.decoding import ReceptiveField
```

Comment:
think I ordered them right this time (next push) but LMK if that's not what you had in mind.